File: algorithms/FooBar/FooBar.swift
var start: Int = 1;
let foo: Int = 5;
let Foo: String = "Foo";
let bar: Int = 7;
let Bar: String = "Bar";
let stop: Int = 100;

for num in start...stop
    {
        if (num % foo == 0 && num % bar == 0)
            {
                print("\(Foo)\(Bar)");
            }
        else if (num % foo == 0)
            {
                print(Foo);
            }
        else if (num % bar == 0)
            {
                print(Bar);
            }
        else
            {
                print(num);
            }
    }

File: algorithms/FooBar/FooBar.java
public class FooBar
{
    public static void main(String[] args) {
        int start, foo, bar, stop, num;
		start = 1;
        foo = 5;
        bar = 7;
        stop = 100;
        String Foo, Bar;
        Foo = "Foo";
        Bar = "Bar";

        for (num = start; num <= stop; num++)
        {
            if (num % foo == 0 && num % bar == 0)
            {
                System.out.println(Foo + Bar);
            }
            else if (num % foo == 0)
            {
                System.out.println(Foo);
            }
            else if (num % bar == 0)
            {
                System.out.println(Bar);
            }
            else
            {
                System.out.println(num);
            }
        }
	}
}
